prow-configurations/config.yaml.tpl

```diff
@@ -12,24 +28,49 @@ presets:
   - name: sa-vm-kyma-integration
     mountPath: /etc/service-account
     readOnly: true
+- labels:
+    preset-sa-gcr-push: "true" # Service account with "Storage Admin" role
+  env:
+    - name: GOOGLE_APPLICATION_CREDENTIALS
+      value: /etc/service-account/sa-gcr-push
+  volumes:
+    - name: sa-gcr-push
+      secret:
+        secretName: sa-gcr-push
+  volumeMounts:
+    - name: sa-gcr-push
+      mountPath: /etc/service-account
```

Comment:
Make mountPath unique for every preset (now it collides with preset-sa-vm-kyma-integration).
Otherwise you can't apply a combination of such labels.
I would suggest to take a fine-grained approach to secrets (lowest possible permissions) and to combine several of them in one job, as necessary for the tasks. For that, mountPaths must be different.

Reply:
Done